examples/pytorch/barlowtwins.py

```diff
@@ -0,0 +1,61 @@
+import torch
+from torch import nn
+import torchvision
+
+import lightly
+from lightly.models.modules import BarlowTwinsProjectionHead
+
+
+class BarlowTwins(nn.Module):
+    def __init__(self, backbone):
+        super().__init__()
+        self.backbone = backbone
+        self.projection_head = BarlowTwinsProjectionHead(512, 2048, 2048)
+
+    def forward(self, x):
+        x = self.backbone(x).flatten(start_dim=1)
+        z = self.projection_head(x)
+        return z
+
+
+resnet = torchvision.models.resnet18()
+backbone = nn.Sequential(*list(resnet.children())[:-1])
+model = BarlowTwins(backbone)
+
+device = "cuda" if torch.cuda.is_available() else "cpu"
+model.to(device)
+
+cifar10 = torchvision.datasets.CIFAR10("datasets/cifar10", download=True)
+dataset = lightly.data.LightlyDataset.from_torch_dataset(cifar10)
+# or create a dataset from a folder containing images or videos:
+# dataset = lightly.data.LightlyDataset("path/to/folder")
+
+collate_fn = lightly.data.ImageCollateFunction(input_size=32)
+
+dataloader = torch.utils.data.DataLoader(
+    dataset,
+    batch_size=256,
+    collate_fn=collate_fn,
+    shuffle=True,
+    drop_last=True,
+    num_workers=8,
+)
+
+criterion = lightly.loss.BarlowTwinsLoss()
+optimizer = torch.optim.SGD(model.parameters(), lr=0.06)
+
+print("Starting Training")
+for epoch in range(10):
+    total_loss = 0
+    for (x0, x1), file_name, label in dataloader:
```

Comment:
if we don't use the label maybe let's just ignore it? Might be more confusing than helpful

Reply:
Ok, removed them.